### PR TITLE
Map kit update

### PR DESCRIPTION
**Pull request by oguzozsoy**

### CODE REVIEW

File: mapkit/build.gradle [hunk 1]
```diff
@@ -2,9 +2,9 @@ apply from: "$project.rootDir/base_sdk.gradle"
 
 dependencies {
     //Huawei Services
-    api 'com.huawei.hms:maps:5.2.0.302'
```
Comment (oguzozsoy) [inline mapkit/build.gradle:5]: please dont update the version

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package com.hms.lib.commonmobileservices.mapkit.model
 
 class CircleOptions {
-    val center: LatLng? = null
+    var center: LatLng? = null
     var radius : Double? = 0.0
 }
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt:26]: Could you please add licenses for CircleOptions class?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/GroundOverlay.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                is HmsGroundOverlay -> groundOverlayImpl.remove()
+                is GmsGroundOverlay -> groundOverlayImpl.remove()
+            }
+            true
+        }catch (e: Exception){
+            e.printStackTrace()
+            false
+        }
+    }
+}
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/GroundOverlay.kt:216]: Could you please add licenses for GroundOverlay class?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/GroundOverlayOptions.kt [hunk 4]
```diff
@@ -0,0 +1,77 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/GroundOverlayOptions.kt:14]: Could you please add licenses for GroundOverlayOptions class?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/LatLngBounds.kt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/LatLngBounds.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TypeAliases.kt [hunk 6]
```diff
@@ -4,4 +4,13 @@ typealias HmsCircle = com.huawei.hms.maps.model.Circle
 typealias GmsCircle = com.google.android.gms.maps.model.Circle
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TypeAliases.kt:17]: Could you please add licenses for CircleOptions class?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt [hunk 7]
```diff
@@ -1,6 +1,6 @@
 package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt:14]: Could you please add licenses?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/Circle.kt [hunk 8]
```diff
@@ -0,0 +1,34 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/Circle.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/Tile.kt [hunk 9]
```diff
@@ -0,0 +1,7 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/Tile.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileOverlay.kt [hunk 10]
```diff
@@ -0,0 +1,33 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileOverlay.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileOverlayOptions.kt [hunk 11]
```diff
@@ -0,0 +1,34 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileOverlayOptions.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileProvider.kt [hunk 12]
```diff
@@ -0,0 +1,7 @@
+package com.hms.lib.commonmobileservices.mapkit.model
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TileProvider.kt:14]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TypeAliases.kt [hunk 13]
```diff
@@ -13,4 +13,10 @@ typealias HmsGroundOverlayOptions = com.huawei.hms.maps.model.GroundOverlayOptio
 typealias GmsGroundOverlayOptions = com.google.android.gms.maps.model.GroundOverlayOptions
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/TypeAliases.kt:26]: Could you please add licenses ?

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/LocationSource.kt [hunk 14]
```diff
@@ -0,0 +1,12 @@
+package com.hms.lib.commonmobileservices.mapkit
```
Comment (omerakkus) [inline mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/LocationSource.kt:14]: Could you please add licenses ?

**Review by omerakkus — CHANGES_REQUESTED** [review]

File: mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt (hunk 7 above)
Comment (omerakkus) [reply mapkit/src/main/java/com/hms/lib/commonmobileservices/mapkit/model/CircleOptions.kt:14]: ok

**Review by omerakkus — APPROVED** [review]

**Pull request merged**